### DISCUSSION TITLE
Feature Request - Add FAIR badges to collections published to GBIF

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#8051] [Ideas]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
_A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]_

No problem - just a new tool available

**Describe what you're trying to accomplish** 
_A clear and concise overview of the goals; why are you asking for this?_

Add a FAIR badge to collections publishing to GBIF - see https://discourse.gbif.org/t/assessing-fairness-of-biodiversity-data-through-badges-and-download-buttons/4246/3

**Describe the solution you'd like**
_How might we accomplish your goals?_

Add this in collection metadata or automatically for any collection being published if possible.

**Describe alternatives you've considered**
_A clear and concise description of any alternative solutions or features you've considered._

Do nothing

**Additional context**
_Add any other context or screenshots about the feature request here._

Continuing collaboration with Jorrit

**Priority**
_Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair._

**Comment by dustymc** [comment]
Is that not implying (guaranteeing?!) that a user can click to download DWC?

**Comment by Jegelewicz** [comment]
No - just that the data is available in that format. If we wanted to supply a link - the DwC files are at Vertnet....
